test/built-ins/Array/prototype/toReversed/this-value-boolean.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2021 Igalia. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.toReversed
+description: >
+  Array.prototype.toReversed throws if the receiver is null or undefined
+info: |
+  Array.prototype.toReversed ( )
+
+  1. Let O be ? ToObject(this value).
+  2. Let len be ? LengthOfArrayLike(O).
+  ...
+features: [change-array-by-copy]
+includes: [compareArray.js]
+---*/
+
+assert.compareArray(Array.prototype.toReversed.call(true), []);
+assert.compareArray(Array.prototype.toReversed.call(false), []);
```

Comment:
Maybe also interesting: add length and indexed properties to `Boolean.prototype`